tools/mo/unit_tests/mo/front/tf/convert_to_pb_test.py

```diff
@@ -154,13 +155,15 @@ def test_meta_format_session_clearing(self):
 
             self.argv.input_meta_graph = os.path.join(tmp_dir, 'model1.meta')
             self.argv.output_dir = tmp_dir
+            self.argv.use_legacy_frontend = true
             path_to_pb = convert_to_pb(self.argv)
             self.assertTrue(os.path.exists(path_to_pb), "The auxiliary .pb is not generated")
             self.assertTrue(os.path.getsize(path_to_pb) != 0, "The auxiliary .pb is empty")
 
             self.argv.input_meta_graph = os.path.join(tmp_dir, 'model2.meta')
             self.argv.output_dir = tmp_dir
             self.argv.input_model = None
+            self.argv.use_legacy_frontend = true
             path_to_pb = convert_to_pb(self.argv)
```

Comment:
not needed. Just expect that `convert_to_pb` returns `None`.

Reply:
can/should I check for legacy frontend? I want to leave this check for legacy frontend.

Reply:
no, the legacy does not use `convert_to_pb` at all. Just leave this test with the same input parameters and check that `convert_to_pb` returns None.